README.md

```diff
@@ -0,0 +1,124 @@
+# Self-trained philosophical word2vec embeddings
+
+## ...obtained out of Hegel's and Kant's corpora.
+
+This project is a part of the means of assessment for the mandatory course 
+"Machine Learning for Natural Language Understanding" of my NLP master's degree
+program at Trier University in the winter semester 22/23. 
+
+## Corpora:
+The texts of great German philosophers Hegel and Kant were chosen as the
+corpora for training the model, namely:
+
+* Kant, Kritik der reinen Vernunft, 1787
+* Kant, Kritik der Urteilskraft, 1790
+* Hegel, Phänomenologie des Geistes, 1807
+* Hegel, Wissenschaft der Logik (Enzyklopädie), 1830
+
+All four texts are taken both in 
+German and in English (from gutenberg.org), so that the first two build Kant’s 
+corpora (`DE_KANT_CORPUS.txt`, `ENG_KANT_CORPUS.txt`) and the other two Hegel’s ones 
+(`DE_HEGEL_CORPUS.txt`, `ENG_HEGEL_CORPUS.txt`).
+
+## Main Tasks:
+1. From-scratch implementation of Skip-Gram architecture: *from raw .txt files to embeddings*
+
+2. Comparison of embeddings' similarity of the most frequently encountered
+philosophical concepts in the corpora of Hegel and Kant (one language, two
+philosophers)
+
+3. Comparison of similarity of English and German embeddings for the same
+philosopher i.e. corpora (two languages, one philosopher)
+
+## Outcomes
+
+### Task 1
+[Top-k-accuracy](https://pytorch.org/docs/stable/generated/torch.topk.html) was used for model evaluation.
+
+Check the results here:
+* [Window size = 3, stop words included.](https://drive.google.com/drive/folders/1TCP6JXWiHNIi86XK45cPJnp_CsjQe0Sc?usp=sharing) (used in Outcomes and Inferences)
```

Comment:
Use [collapsed markdown ](https://codex.so/collapsed-block-for-github-markdown) instead of links. And put pictures in the readme (upload images in git and then add a link to them from readme)